frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetAccessManagement/ClusterSetAccessManagement.tsx

```diff
@@ -354,7 +360,7 @@ function AddUsersModal(props: {
                             &nbsp;
                             <AcmSelect
                                 id="role"
-                                maxHeight="6em"
+                                maxHeight="10em"
```

Comment:
@KevinFCormier 6em gets us to about 1.5 visible select elements, no reason to skimp here as Patter Fly guidelines do suggest 7 or less items be visible when select dropdown is active, so I extended the length to cover both elements in the dropdown, I might add more units but this list ("User role") does not look like it will change in size often...unless cluster set roles are something users can add to. In which case I would probably want to increase this to **25-30em**. I do wonder why we use **em** here too...

Reply:
I don't anticipate adding more options, and to be honest, I'm not sure what the history behind using **em** is, but the last case of this we used `maxHeight="18em"`

https://github.com/open-cluster-management/console/blob/main/frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx#L545